CONTRIBUTING.md

```diff
@@ -21,7 +21,10 @@ _If you're looking into contributing to the docs, follow the [instructions](#bui
 ### Prerequisites
 
 - git
-- node.js
+- node.js (run `npm pkg get engines.node` for the right version)
+- pnpm
```

Comment:
How do you feel about just linking to their [installation page](https://pnpm.io/installation)? That way we leave it up to the pnpm team to document what's the ideal way of using their tool?

Reply:
Sure, that is a good idea, however this page is about setting up (a typical set up) for Toolpad, and I believe, since the typical usage with node is npm, there should at least be a pointer in how Toolpad typically uses the package manager, moreover because corepack is experimental, but preferred

Reply:
update; I don't think it is experimental from node 20 onwards

```sh
❯ npm pkg get engines.node
">=20"
  ~/git/hub/toolpad   feature/pnpm-docs      
```